src/components/ProductsPage/ProductsSort.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+interface Props {
+  sortByOption: string;
+  perPage: string;
+  setParams: (event: string, option: string) => (void);
```

Comment:
```suggestion
  setParams: (value: string, option: string) => (void);
```
это ведь value а не ивент